src/app/frontend/podlist/podcard_component.js

```diff
@@ -82,7 +83,59 @@ export class PodCardController {
    * @export
    */
   isFailed() {
-    return this.pod.podStatus.podPhase === 'Failed' || this.hasWarnings();
+    if (this.pod.podStatus.podPhase === 'Failed') {
```

Comment:
For some reason now none of my pods are "Green". Everything is pending or failed.